src/applications/mhv/secure-messaging/tests/components/ThreadListComponents/ThreadsList.unit.spec.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderWithStoreAndRouter } from '@department-of-veterans-affairs/platform-testing/react-testing-library-helpers';
+import ThreadsList from '../../../components/ThreadList/ThreadsList';
+import inbox from '../../fixtures/folder-inbox-response.json';
+import listOfThreads from '../../fixtures/thread-list-response.json';
+import reducers from '../../../reducers';
+
+describe('Thread List component', () => {
+  const initialState = {
+    sm: {
+      folders: {},
+      threads: [],
+    },
+  };
+
+  const setPageNum = () => 1;
+  const setSortOrder = () => 'ASC';
+  const setSortBy = () => 'SENDER_NAME';
+  const handleThreadApiCall = () => {
+    return listOfThreads;
+  };
+  const threadsPerPage = 10;
+  const folderId = 0;
+  const pageNum = 1;
+  let screen;
+
+  const setup = () => {
+    return renderWithStoreAndRouter(
+      <ThreadsList
+        threadList={listOfThreads}
+        folder={inbox}
+        folderId={folderId}
+        setPageNum={setPageNum}
+        pageNum={pageNum}
+        setSortOrder={setSortOrder}
+        setSortBy={setSortBy}
+        handleThreadApiCall={handleThreadApiCall}
+        threadsPerPage={threadsPerPage}
+      />,
+      {
+        path: `/inbox`,
+        state: initialState,
+        reducers,
+      },
+    );
+  };
+  it('renders without errors', () => {
+    screen = setup();
+    expect(screen);
+  });
+  it('renders list of threads', async () => {
+    screen = setup();
+
+    const renderedThreads = await screen.getAllByTestId('thread-list-item');
+    expect(renderedThreads.length).to.equal(13);
+  });
+});
```

Comment:
I would add test coverage for a list with no threads

Reply:
Thanks for that recommendation, it prompted me to create a flow for that scenario, since we are not getting back an empty list from the api call, instead we get a 400 bad request.